docs/vtex-io/Store-Framework-Apps/vtex-b2b-suite/vtex-b2b-organizations.md

```diff
@@ -526,7 +526,7 @@ Users with the **Sales Admin** role may add, edit, or remove users with any of t
 
 In telesales, assisted sales, or customer support scenarios where the B2B customer needs help from a salesperson to navigate the store or place orders, it is common to use impersonation.
 
-In the **B2B Organizations** app, only users with **Sales** roles or the **Organization Admin** role can impersonate another user.
+In the **B2B Organizations** app, users with one of the following permissions may impersonate another user: **impersonate-users-all**, **impersonate-users-organization** or **impersonate-users-costcenter**.
```

Comment:
The word "permissions" is vague. However, I could not find these permissions in the [LM resource article](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3), so I am guessing there is another permission system related to the b2b suite. So I recommend:
- **_If it really is another permission system_**, make it clear where users can configure this and link to an article about it.
- **_If these are LM resources_**, please [document these resources](https://www.notion.so/vtexhandbook/LM-docs-process-665424ddf4a2406784fbe0225f1e4600) and also link the [LM resource article](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) here.

Reply:
I rephrased the text to make it clearer that the permissions belong to the Storefront Permission.